server/datastore/mysql/software.go

```diff
@@ -659,6 +659,42 @@ func (si *softwareIterator) Next() bool {
 	return si.rows.Next()
 }
 
+func (ds *Datastore) ListSoftwareBySourceIter(
+	ctx context.Context,
+	sources []string,
+) (fleet.SoftwareIterator, error) {
+	if len(sources) == 0 {
+		return nil, errors.New("please provide at least one source")
+	}
+
+	var err error
+	var args []interface{}
+
+	stmt := `SELECT 
+		s.id, 
+		s.name,
+		s.version,
+		s.bundle_identifier,
+		s.release,
+		s.vendor,
+		s.arch,
+		s.source 
+	FROM software s
```

Comment:
When running this on my mac I get:  `no such table: software`, is this table added later in the PR?